misc/format.py

```diff
@@ -23,7 +23,7 @@ def main():
                 files.append(os.path.join(dirpath, f))
 
     for filename in files:
-        subprocess.call('clang-format --style=file:{0}/misc/.clang-format -i {1}'.format(root_path, filename), shell=True)
+        subprocess.call('clang-format --style=file:./misc/.clang-format -i {0}'.format(filename), shell=True)
```

Comment:
Does this fix the relative path issues now with the workflow?

Reply:
No. That still does not work.